src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1230,6 +1230,18 @@ public function validateIpv6($attribute, $value)
         return filter_var($value, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) !== false;
     }
 
+    /**
+     * Validate that an attribute is a valid MAC address.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateMacAddress($attribute, $value)
```

Comment:
Maybe name the validation rule just "mac" to be consistent with "email" and "ip" for example?

Reply:
I don't think it will **understandable**, **clear** enough, it could be consistent with "accepted_if", "require_if"